dojo/finding/views.py

```diff
@@ -727,6 +731,79 @@ def process_form(self, request: HttpRequest, finding: Finding, context: dict):
 
         return request, False
 
+
+    def extract_cves(self, description: str):
```

Comment:
Function does ont seem to use `self`, maybe we should move it inside `DataStore` instead.

Reply:
https://github.com/iago-r/django-DefectDojo/pull/8